tembo-stacks/src/stacks/specs/oltp.yaml

```diff
@@ -38,9 +38,23 @@ postgres_config:
 trunk_installs:
   - name: pg_stat_statements
     version: 1.10.0
+  - name: parquet_s3_fdw
```

Comment:
i didnt think we wanted this in the OLTP stack by default

Reply:
Ok, I will remove pg_tier from OLTP stack.